src/radio/utils/PlaylistParser.js

```diff
@@ -0,0 +1,59 @@
+/* @flow */
+
+import https from 'https';
+import querystring from 'querystring';
+import type {ConfigOptions} from '../..';
+
+const getUrl = 'https://www.googleapis.com/youtube/v3/playlistItems?';
+const youtubeUrl = 'https://www.youtube.com/watch?v=';
+
+export default function parsePlaylist(
+  id: string,
+  config: string,
+  token?: string,
```

Comment:
i think you missed a comment i added late but i think this should be called `pageToken`
https://github.com/Uppun/lilianne/pull/7#discussion_r170440170